realm-jni/src/io_realm_internal_SharedGroup.cpp

```diff
@@ -150,21 +150,21 @@ JNIEXPORT jlong JNICALL Java_io_realm_internal_SharedGroup_nativeBeginImplicit
 }
 
 JNIEXPORT void JNICALL Java_io_realm_internal_SharedGroup_nativeAdvanceRead
-(JNIEnv *env, jobject, jlong native_ptr)
+(JNIEnv *env, jobject, jlong native_ptr, jlong native_replication_ptr)
```

Comment:
native_replication_ptr - should't that now be named something like native_ch_ptr or native_client_history_ptr? (here and other places)